pipeline/splicing/calcVarPriors.py

```diff
@@ -139,22 +168,314 @@ def getVarType(variant):
+    inExon = varInExon(variant)
+    if varGene == "BRCA1":
+        for domain in brca1CIDomains[boundaries].keys():
+            domainStart = brca1CIDomains[boundaries][domain]["domStart"]
+            domainEnd = brca1CIDomains[boundaries][domain]["domEnd"]
+            if varGenPos <= domainStart and varGenPos >= domainEnd and inExon == True:
+                return True
+    elif varGene == "BRCA2":
+        for domain in brca2CIDomains[boundaries].keys():
+            domainStart = brca2CIDomains[boundaries][domain]["domStart"]
+            domainEnd = brca2CIDomains[boundaries][domain]["domEnd"]
+            if varGenPos >= domainStart and varGenPos <= domainEnd and inExon == True:
+                return True
+    return False
+                
+
+def varInGreyZone(variant):
+    '''
+    Given a variant, determines if variant is before, in, or after the grey zone
+    Returns false if variant is before the grey zone
+    Returns "grey_zone_variant" if variant is in the grey zone
+    Return "after_grey_zone_variant" if variant is after the grey zone
+    '''
+    varGenPos = int(variant["Pos"])
+    varGene = variant["Gene_Symbol"]
+    inUTR = varInUTR(variant)
+    if varGene == "BRCA2":
+        greyZoneStart = greyZones[varGene]["greyZoneStart"]
+        greyZoneEnd = greyZones[varGene]["greyZoneEnd"]
+        if varGenPos >= greyZoneStart and varGenPos <= greyZoneEnd:
+            return "grey_zone_variant"
+        elif varGenPos > greyZoneEnd and inUTR == False:
+            return "after_grey_zone_variant"
+    return False
+                
+
+def getVarLocation(variant, boundaries):
+    '''
+    Given a variant, returns the variant location as below
+    Second argument is for CI domain boundaries (PRIORS or ENIGMA)
+    '''
+    varOutBounds = varOutsideBoundaries(variant)
+    if varOutBounds == True:
+        return "outside_transcript_boundaries_variant"
+    inExon = varInExon(variant)
+    if inExon == True:
+        inCiDomain = varInCiDomain(variant, boundaries)
+        if inCiDomain == True:
+            inSpliceDonor = varInSpliceDonor(variant)
+            if inSpliceDonor == True:
+                return "CI_splice_donor_variant"
+            inSpliceAcceptor = varInSpliceAcceptor(variant)
+            if inSpliceAcceptor == True:
+                return "CI_splice_acceptor_variant"
+            return "CI_domain_variant"
+        inSpliceDonor = varInSpliceDonor(variant)
+        if inSpliceDonor == True:
+            return "splice_donor_variant"
+        inSpliceAcceptor = varInSpliceAcceptor(variant)
+        if inSpliceAcceptor == True:
+            return "splice_acceptor_variant"
+        inGreyZone = varInGreyZone(variant)
+        if inGreyZone != False:
+            return inGreyZone
+        inUTR = varInUTR(variant)
+        if inUTR == True:
+            return "UTR_variant"
+        return "exon_variant"
+    else:    
+        inSpliceDonor = varInSpliceDonor(variant)
```

Comment:
again, maybe just call it at the very beginning along with inSpliceAcceptor to reduce the number of lines in this method?